### PR TITLE
main/perl-inline: fix

### PR DESCRIPTION
**Pull request by maxice8**
- Fix wrong license
- use $() instead of backticks ``

@rnalrd

### CODE REVIEW

**Comment by maxice8** [comment]
Most packages use both

**Comment by rnalrd** [comment]
I know, I'm asking opinions on what is correct: i think we should pick one. Other non-perl packages list multiple licenses because different pieces of code use different licenses. This is not the case.

**Comment by maxice8** [comment]
Well we can extend our usage of SPDX and make use of the `OR` operator

GPL-1.0-or-later OR Artistic-1.0-Perl

> Other non-perl packages list multiple licenses because different pieces of code use different licenses.

Not always true, see dual-licensing. I am in favor of  using SPDX's `OR` to show it. If they have different pieces of code they should use `AND`

File: main/perl-inline/APKBUILD [hunk 1]
```diff
@@ -17,12 +17,12 @@ builddir="$srcdir"/$_pkgname-$pkgver
 
 prepare() {
 	default_prepare
-	export CFLAGS=`perl -MConfig -E 'say $Config{ccflags}'`
+	export CFLAGS="$(perl -MConfig -E 'say $Config{ccflags}')"
```
Comment (fabled) [inline main/perl-inline/APKBUILD]: Currently `apkbuild-cpan` does not include quotation marks in the new templates. Perhaps use it to regenerate this? Or update also it to contain the quotes. In any case, expansion like this should not strictly need the quotes, and that's why I omitted them. Thoughts?

**Comment by maxice8** [comment]
On Mon, May 27, 2019 at 11:59:37PM -0700, Timo Teräs wrote:
> fabled commented on this pull request.
> 
> 
> 
> > @@ -17,12 +17,12 @@ builddir="$srcdir"/$_pkgname-$pkgver
>  
>  prepare() {
>  	default_prepare
> -	export CFLAGS=`perl -MConfig -E 'say $Config{ccflags}'`
> +	export CFLAGS="$(perl -MConfig -E 'say $Config{ccflags}')"
> 
> Currently `apkbuild-cpan` does not include quotation marks in the new templates. Perhaps use it to regenerate this? Or update also it to contain the quotes. In any case, expansion like this should not strictly need the quotes, and that's why I omitted them. Thoughts?
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/alpinelinux/aports/pull/7433#pullrequestreview-242477799

Fixed

**Pull request closed without merging**